toolz/itertoolz.py

```diff
@@ -15,6 +14,9 @@
            'join', 'tail', 'diff', 'topk', 'peek')
 
 
+_no_default = object()  # sentinel
```

Comment:
This shouldn't pose a problem for `cytoolz`.  Note that we also define `no_default` in `toolz.utils`, so we should probably just clean it up there.

Also, a benefit of using `'__no__default__'` as before is that it looks nice(-ish) when inspecting the function.  Perhaps we can make a `_no_default` sentinel value in utils, such as:
```python
class _NoDefault(object):
    def __str__(self):
        return '<object no_default>'
    __repr__ = __str__

_no_default = _NoDefault()
```